### ISSUE TITLE
 Unable to see signature help: Error running timer: (void-function lsp-cur-line-diagnostics)

### ISSUE THREAD

**Issue by sp3ctum** [emacs-lsp/lsp-mode#819]
**Describe the bug**
Getting an error when coding in a python file.

**To Reproduce**
- open a python file
- start `(lsp)` and `(lsp-ui-mode t)`
- move point to a line with some code

**Expected behavior**
Expected to see the function signature in an lsp-ui-sideline popup.

**Which Language Server did you use**
lsp-python

**OS**
Fedora Linux 28

**Error callstack**

Backtrace:
```
Debugger entered--Lisp error: (void-function lsp-cur-line-diagnostics)
  (lsp-cur-line-diagnostics)
  (list :diagnostics (lsp-cur-line-diagnostics))
  (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics)))
  (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))
  (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params)))
  (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive)
  (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive))
  (if (and lsp-ui-sideline-show-code-actions (or (lsp--capability "codeActionProvider") (lsp--registered-capability "textDocument/codeAction"))) (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive)))
  (save-excursion (setq lsp-ui-sideline--occupied-lines nil lsp-ui-sideline--tag tag lsp-ui-sideline--last-width (window-text-width)) (if lsp-ui-sideline-show-diagnostics (progn (lsp-ui-sideline--diagnostics bol eol))) (if (and lsp-ui-sideline-show-code-actions (or (lsp--capability "codeActionProvider") (lsp--registered-capability "textDocument/codeAction"))) (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive))) (if (and lsp-ui-sideline-show-hover (lsp--capability "hoverProvider")) (progn (let ((symbols)) (goto-char eol) (while (and (> (point) bol) (progn (forward-symbol -1) (>= (point) bol))) (let* ((symbol (thing-at-point 'symbol t)) (bounds (bounds-of-thing-at-point 'symbol)) (parsing-state (syntax-ppss)) (in-string (nth 3 parsing-state)) (outside-comment (eq (nth 4 parsing-state) nil))) (if (and symbol (not in-string) outside-comment) (progn (setq symbols (cons (list symbol tag bounds (list :line (1- line-widen) :character (- (point) bol))) symbols)))))) (let ((list symbols) (it-index 0)) (while list (let ((it (car list))) (let ((input0 it) (input1 it-index)) (let* ((--dash-source-53-- input0) (symbol (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (tag (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (bounds (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (position (car --dash-source-53--)) (index input1)) (lsp--send-request-async (lsp--make-request "textDocument/hover" (list :textDocument doc-id :position position)) (function (lambda (info) (if (eq index 0) (progn (let ((--dolist-tail-- lsp-ui-sideline--ovs)) (while --dolist-tail-- (let ((ov (car --dolist-tail--))) (if (eq (overlay-get ov 'kind) 'info) (progn (setq lsp-ui-sideline--occupied-lines (delq (overlay-get ov 'position) lsp-ui-sideline--occupied-lines)) (delete-overlay ov))) (setq --dolist-tail-- (cdr --dolist-tail--))))))) (if info (progn (lsp-ui-sideline--push-info symbol tag bounds info bol eol))))))))) (setq it-index (1+ it-index)) (setq list (cdr list))))))))
  (let ((eol (line-end-position)) (bol (line-beginning-position)) (tag (lsp-ui-sideline--calculate-tag)) (line-widen (save-restriction (widen) (line-number-at-pos))) (doc-id (list :uri (lsp--buffer-uri)))) (save-excursion (setq lsp-ui-sideline--occupied-lines nil lsp-ui-sideline--tag tag lsp-ui-sideline--last-width (window-text-width)) (if lsp-ui-sideline-show-diagnostics (progn (lsp-ui-sideline--diagnostics bol eol))) (if (and lsp-ui-sideline-show-code-actions (or (lsp--capability "codeActionProvider") (lsp--registered-capability "textDocument/codeAction"))) (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive))) (if (and lsp-ui-sideline-show-hover (lsp--capability "hoverProvider")) (progn (let ((symbols)) (goto-char eol) (while (and (> (point) bol) (progn (forward-symbol -1) (>= (point) bol))) (let* ((symbol (thing-at-point 'symbol t)) (bounds (bounds-of-thing-at-point 'symbol)) (parsing-state (syntax-ppss)) (in-string (nth 3 parsing-state)) (outside-comment (eq (nth 4 parsing-state) nil))) (if (and symbol (not in-string) outside-comment) (progn (setq symbols (cons (list symbol tag bounds (list :line (1- line-widen) :character (- (point) bol))) symbols)))))) (let ((list symbols) (it-index 0)) (while list (let ((it (car list))) (let ((input0 it) (input1 it-index)) (let* ((--dash-source-53-- input0) (symbol (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (tag (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (bounds (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (position (car --dash-source-53--)) (index input1)) (lsp--send-request-async (lsp--make-request "textDocument/hover" (list :textDocument doc-id :position position)) (function (lambda (info) (if (eq index 0) (progn (let ((--dolist-tail-- lsp-ui-sideline--ovs)) (while --dolist-tail-- (let ((ov (car --dolist-tail--))) (if (eq (overlay-get ov 'kind) 'info) (progn (setq lsp-ui-sideline--occupied-lines (delq (overlay-get ov 'position) lsp-ui-sideline--occupied-lines)) (delete-overlay ov))) (setq --dolist-tail-- (cdr --dolist-tail--))))))) (if info (progn (lsp-ui-sideline--push-info symbol tag bounds info bol eol))))))))) (setq it-index (1+ it-index)) (setq list (cdr list)))))))))
  (progn (let ((eol (line-end-position)) (bol (line-beginning-position)) (tag (lsp-ui-sideline--calculate-tag)) (line-widen (save-restriction (widen) (line-number-at-pos))) (doc-id (list :uri (lsp--buffer-uri)))) (save-excursion (setq lsp-ui-sideline--occupied-lines nil lsp-ui-sideline--tag tag lsp-ui-sideline--last-width (window-text-width)) (if lsp-ui-sideline-show-diagnostics (progn (lsp-ui-sideline--diagnostics bol eol))) (if (and lsp-ui-sideline-show-code-actions (or (lsp--capability "codeActionProvider") (lsp--registered-capability "textDocument/codeAction"))) (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive))) (if (and lsp-ui-sideline-show-hover (lsp--capability "hoverProvider")) (progn (let ((symbols)) (goto-char eol) (while (and (> (point) bol) (progn (forward-symbol -1) (>= (point) bol))) (let* ((symbol (thing-at-point 'symbol t)) (bounds (bounds-of-thing-at-point 'symbol)) (parsing-state (syntax-ppss)) (in-string (nth 3 parsing-state)) (outside-comment (eq (nth 4 parsing-state) nil))) (if (and symbol (not in-string) outside-comment) (progn (setq symbols (cons (list symbol tag bounds (list :line (1- line-widen) :character (- (point) bol))) symbols)))))) (let ((list symbols) (it-index 0)) (while list (let ((it (car list))) (let ((input0 it) (input1 it-index)) (let* ((--dash-source-53-- input0) (symbol (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (tag (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (bounds (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (position (car --dash-source-53--)) (index input1)) (lsp--send-request-async (lsp--make-request "textDocument/hover" (list :textDocument doc-id :position position)) (function (lambda (info) (if (eq index 0) (progn (let ((--dolist-tail-- lsp-ui-sideline--ovs)) (while --dolist-tail-- (let ((ov (car --dolist-tail--))) (if (eq (overlay-get ov 'kind) 'info) (progn (setq lsp-ui-sideline--occupied-lines (delq (overlay-get ov 'position) lsp-ui-sideline--occupied-lines)) (delete-overlay ov))) (setq --dolist-tail-- (cdr --dolist-tail--))))))) (if info (progn (lsp-ui-sideline--push-info symbol tag bounds info bol eol))))))))) (setq it-index (1+ it-index)) (setq list (cdr list))))))))))
  (if buffer-file-name (progn (let ((eol (line-end-position)) (bol (line-beginning-position)) (tag (lsp-ui-sideline--calculate-tag)) (line-widen (save-restriction (widen) (line-number-at-pos))) (doc-id (list :uri (lsp--buffer-uri)))) (save-excursion (setq lsp-ui-sideline--occupied-lines nil lsp-ui-sideline--tag tag lsp-ui-sideline--last-width (window-text-width)) (if lsp-ui-sideline-show-diagnostics (progn (lsp-ui-sideline--diagnostics bol eol))) (if (and lsp-ui-sideline-show-code-actions (or (lsp--capability "codeActionProvider") (lsp--registered-capability "textDocument/codeAction"))) (progn (lsp--send-request-async (lsp--make-request "textDocument/codeAction" (if (equal lsp-ui-sideline-update-mode 'line) (list :textDocument doc-id :range (lsp--range (lsp--point-to-position bol) (lsp--point-to-position eol)) :context (list :diagnostics (lsp-cur-line-diagnostics))) (lsp--text-document-code-action-params))) (function (lambda (actions) (lsp-ui-sideline--code-actions actions bol eol))) 'alive))) (if (and lsp-ui-sideline-show-hover (lsp--capability "hoverProvider")) (progn (let ((symbols)) (goto-char eol) (while (and (> (point) bol) (progn (forward-symbol -1) (>= (point) bol))) (let* ((symbol (thing-at-point 'symbol t)) (bounds (bounds-of-thing-at-point 'symbol)) (parsing-state (syntax-ppss)) (in-string (nth 3 parsing-state)) (outside-comment (eq (nth 4 parsing-state) nil))) (if (and symbol (not in-string) outside-comment) (progn (setq symbols (cons (list symbol tag bounds (list :line (1- line-widen) :character (- (point) bol))) symbols)))))) (let ((list symbols) (it-index 0)) (while list (let ((it (car list))) (let ((input0 it) (input1 it-index)) (let* ((--dash-source-53-- input0) (symbol (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (tag (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (bounds (car-safe (prog1 --dash-source-53-- (setq --dash-source-53-- (cdr --dash-source-53--))))) (position (car --dash-source-53--)) (index input1)) (lsp--send-request-async (lsp--make-request "textDocument/hover" (list :textDocument doc-id :position position)) (function (lambda (info) (if (eq index 0) (progn (let ((--dolist-tail-- lsp-ui-sideline--ovs)) (while --dolist-tail-- (let ((ov (car --dolist-tail--))) (if (eq (overlay-get ov 'kind) 'info) (progn (setq lsp-ui-sideline--occupied-lines (delq (overlay-get ov 'position) lsp-ui-sideline--occupied-lines)) (delete-overlay ov))) (setq --dolist-tail-- (cdr --dolist-tail--))))))) (if info (progn (lsp-ui-sideline--push-info symbol tag bounds info bol eol))))))))) (setq it-index (1+ it-index)) (setq list (cdr list)))))))))))
  lsp-ui-sideline--run()
  (progn (lsp-ui-sideline--run))
  (if (equal buf (current-buffer)) (progn (lsp-ui-sideline--run)))
  (closure ((buf . #<buffer joukahainen_adjective_tosi.py>) t) nil (if (equal buf (current-buffer)) (progn (lsp-ui-sideline--run))))()
  apply((closure ((buf . #<buffer joukahainen_adjective_tosi.py>) t) nil (if (equal buf (current-buffer)) (progn (lsp-ui-sideline--run)))) nil)
  timer-event-handler([t 0 0 200000 nil (closure ((buf . #<buffer joukahainen_adjective_tosi.py>) t) nil (if (equal buf (current-buffer)) (progn (lsp-ui-sideline--run)))) nil idle 0])
```

lsp-log:

```
Command "pyls" is present on the path.
Found the following clients for /home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py: (server-id pyls, priority -1)
The following clients were selected based on priority: (server-id pyls, priority -1)
>>> pyls:26831 status:starting(async)
Content-Length: 3111

{
  "jsonrpc": "2.0",
  "method": "initialize",
  "params": {
    "processId": 25022,
    "rootPath": "/home/mvilpas/git/plover-finnish/",
    "rootUri": "file:///home/mvilpas/git/plover-finnish/",
    "capabilities": {
      "workspace": {
        "workspaceEdit": {
          "documentChanges": true,
          "resourceOperations": [
            "create",
            "rename",
            "delete"
          ]
        },
        "applyEdit": true,
        "symbol": {
          "symbolKind": {
            "valueSet": [
              1,
              2,
              3,
              4,
              5,
              6,
              7,
              8,
              9,
              10,
              11,
              12,
              13,
              14,
              15,
              16,
              17,
              18,
              19,
              20,
              21,
              22,
              23,
              24,
              25,
              26
            ]
          }
        },
        "executeCommand": {
          "dynamicRegistration": false
        },
        "didChangeWatchedFiles": {
          "dynamicRegistration": true
        },
        "workspaceFolders": true
      },
      "textDocument": {
        "declaration": {
          "linkSupport": true
        },
        "definition": {
          "linkSupport": true
        },
        "implementation": {
          "linkSupport": true
        },
        "typeDefinition": {
          "linkSupport": true
        },
        "synchronization": {
          "willSave": true,
          "didSave": true,
          "willSaveWaitUntil": true
        },
        "documentSymbol": {
          "symbolKind": {
            "valueSet": [
              1,
              2,
              3,
              4,
              5,
              6,
              7,
              8,
              9,
              10,
              11,
              12,
              13,
              14,
              15,
              16,
              17,
              18,
              19,
              20,
              21,
              22,
              23,
              24,
              25,
              26
            ]
          },
          "hierarchicalDocumentSymbolSupport": true
        },
        "formatting": {
          "dynamicRegistration": true
        },
        "codeAction": {
          "dynamicRegistration": true
        },
        "completion": {
          "completionItem": {
            "snippetSupport": true
          }
        },
        "signatureHelp": {
          "signatureInformation": {
            "parameterInformation": {
              "labelOffsetSupport": true
            }
          }
        },
        "documentLink": {
          "dynamicRegistration": true
        },
        "hover": {
          "contentFormat": [
            "plaintext",
            "markdown"
          ]
        },
        "foldingRange": {
          "dynamicRegistration": true,
          "rangeLimit": null,
          "lineFoldingOnly": null
        }
      }
    },
    "initializationOptions": null
  },
  "id": 1
}

Command "pyls" is present on the path.
Found the following clients for /home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py: (server-id pyls, priority -1)
The following clients were selected based on priority: (server-id pyls, priority -1)
<<<< pyls:26831 status:starting
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "capabilities": {
      "codeActionProvider": true,
      "codeLensProvider": {
        "resolveProvider": false
      },
      "completionProvider": {
        "resolveProvider": false,
        "triggerCharacters": [
          "."
        ]
      },
      "documentFormattingProvider": true,
      "documentHighlightProvider": true,
      "documentRangeFormattingProvider": true,
      "documentSymbolProvider": true,
      "definitionProvider": true,
      "executeCommandProvider": {
        "commands": []
      },
      "hoverProvider": true,
      "referencesProvider": true,
      "renameProvider": true,
      "signatureHelpProvider": {
        "triggerCharacters": [
          "(",
          ","
        ]
      },
      "textDocumentSync": 2,
      "experimental": null
    }
  }
}
>>> pyls:26831(async)
Content-Length: 69

{
  "jsonrpc": "2.0",
  "method": "initialized",
  "params": {
  }
}

>>> pyls:26831(async)
Content-Length: 987

{
  "jsonrpc": "2.0",
  "method": "textDocument/didOpen",
  "params": {
    "textDocument": {
      "uri": "file:///home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py",
      "languageId": "python",
      "version": 0,
      "text": "from .. import base\nfrom .adjective import Adjective, inflect_as, inflect_with\n\n\ndef joukahainen_adjective_tosi(word, gradation_fn = base.identity) -> Adjective:\n    # the word täysi only\n\n    (täy, s, i) = base.reverse_parse(gradation_fn(word), base.root_and_cv_ending)\n\n    # The example word is \"täysi;adjective-tosi\"\n    täydempi = täy + \"dempi\"\n    täysin   = täy + s + i + \"n\"\n\n    return Adjective(positive    = inflect_with(word, \"tosi\", gradation_fn),\n                     comparative = inflect_as(täydempi, \"suurempi\"),\n                     superlative = inflect_as(täysin, \"sisin\"))\n"
    }
  }
}

>>> pyls:26831(async)
Content-Length: 1827

{
  "jsonrpc": "2.0",
  "method": "workspace/didChangeConfiguration",
  "params": {
    "settings": {
      "pyls": {
        "plugins.jedi_completion.enabled": true,
        "plugins.jedi_definition.follow_imports": true,
        "configurationSources": [
          "pycodestyle"
        ],
        "plugins.jedi_completion.enabled": true,
        "plugins.jedi_definition.enabled": true,
        "plugins.jedi_definition.follow_imports": null,
        "plugins.jedi_definition.follow_builtin_imports": null,
        "plugins.jedi_hover.enabled": true,
        "plugins.jedi_references.enabled": true,
        "plugins.jedi_signature_help.enabled": null,
        "plugins.jedi_symbols.enabled": null,
        "plugins.jedi_symbols.all_scopes": true,
        "plugins.mccabe.enabled": null,
        "plugins.mccabe.threshold": 15,
        "plugins.preload.enabled": true,
        "plugins.preload.modules": null,
        "plugins.pycodestyle.enabled": true,
        "plugins.pycodestyle.exclude": null,
        "plugins.pycodestyle.filename": null,
        "plugins.pycodestyle.select": null,
        "plugins.pycodestyle.ignore": null,
        "plugins.pycodestyle.hangClosing": null,
        "plugins.pycodestyle.maxLineLength": null,
        "plugins.pydocstyle.enabled": null,
        "plugins.pydocstyle.convention": null,
        "plugins.pydocstyle.addIgnore": null,
        "plugins.pydocstyle.addSelect": null,
        "plugins.pydocstyle.ignore": null,
        "plugins.pydocstyle.select": null,
        "plugins.pydocstyle.match": "(?!test_).*\\.py",
        "plugins.pydocstyle.matchDir": null,
        "plugins.pyflakes.enabled": true,
        "plugins.rope_completion.enabled": true,
        "plugins.yapf.enabled": true,
        "rope.extensionModules": null,
        "rope.ropeFolder": null
      }
    }
  }
}

<<<< pyls:26831
{
  "jsonrpc": "2.0",
  "method": "textDocument/publishDiagnostics",
  "params": {
    "uri": "file:///home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py",
    "diagnostics": [
      {
        "source": "mypy",
        "range": {
          "start": {
            "line": 0,
            "character": 1
          },
          "end": {
            "line": 0,
            "character": 5
          }
        },
        "message": "No parent module -- cannot perform relative import",
        "severity": 1
      }
    ]
  }
}
>>> pyls:26831(async)
Content-Length: 362

{
  "jsonrpc": "2.0",
  "method": "textDocument/hover",
  "params": {
    "textDocument": {
      "uri": "file:///home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py"
    },
    "position": {
      "line": 15,
      "character": 24
    }
  },
  "id": 2
}

<<<< pyls:26831
{
  "jsonrpc": "2.0",
  "id": 2,
  "result": {
    "contents": ""
  }
}
>>> pyls:26831(async)
Content-Length: 362

{
  "jsonrpc": "2.0",
  "method": "textDocument/hover",
  "params": {
    "textDocument": {
      "uri": "file:///home/mvilpas/git/plover-finnish/stroke_dictionary_creator/stroke_dictionary_creator/inflection/roots/inflection_types/adjectives/joukahainen_adjective_tosi.py"
    },
    "position": {
      "line": 14,
      "character": 24
    }
  },
  "id": 3
}

<<<< pyls:26831
{
  "jsonrpc": "2.0",
  "id": 3,
  "result": {
    "contents": ""
  }
}
Buffer switched - ignoring reponse. Method textDocument/hover
```

**Comment by sp3ctum** [comment]
This error started appearing when I updated spacemacs to the newest develop version, and updated all my packages  too. Before that, all things worked well.

I looked at this bug: https://github.com/emacs-lsp/lsp-mode/issues/801 but I think this may be a bit different. Any advice on how to work around this?

Here are my lsp related versions:
```
elpa/26.1/develop/company-lsp-20190505.615/
elpa/26.1/develop/helm-lsp-20190423.548/
elpa/26.1/develop/lsp-java-20190505.2003/
elpa/26.1/develop/lsp-mode-20190321.1743/
elpa/26.1/develop/lsp-treemacs-20190503.956/
elpa/26.1/develop/lsp-ui-20190428.1045/
```

Oh, and for what it's worth, I can see errors from pyflakes just fine (syntax errors work).
